include/irr/asset/format/encodePixels.h

```diff
@@ -7,9 +7,43 @@
 #include "irr/core/core.h"
 #include "irr/asset/format/EFormat.h"
 
-
 namespace irr { namespace video
 {	
+
+	template<typename T>
+	inline uint64_t getRangeValueOfVariable(bool maxValue = true)
```

Comment:
don't we have this function already in terms of `E_FORMAT` @Crisspl ?

Reply:
ok @AnastaZIuk answer accepted.

Reply:
so is there such?

Reply:
nope not for this.

Reply:
make it `constexpr inline uint64_t`

Reply:
sure, to fix